### PR TITLE
Doc update

### PR DESCRIPTION
**Pull request by nchampollion**
Feel free to take the suggestions into account or not. I continue with USING OGGM section later.

Closes https://github.com/OGGM/oggm/issues/###

- [ ] Tests added/passed
- [ ] Fully documented
- [ ] Entry in `whats-new.rst`

### CODE REVIEW

File: docs/flowlines.rst [hunk 1]
```diff
@@ -147,7 +147,10 @@ the glacier outlines or (ii) the catchment boundaries:
 
 Then, these geometrical widths are corrected so that the altitude-area
 distribution of the "flowline-glacier" is as close as possible as the actual
-distribution of the glacier using its full 2D geometry:
+distribution of the glacier using its full 2D geometry. This correction is responsible
+for the large widths of the main red flowline in its upper part.
+This width increase comes from the two northern glacier sub-catchments that are not
+included in the main flowline:
```
Comment (fmaussion) [inline docs/flowlines.rst:153]:
```suggestion
included in the main flowline.
```

File: docs/flowlines.rst [hunk 2]
```diff
@@ -163,10 +166,10 @@ but it's close enough.
 Elevation bands flowlines
 -------------------------
 
-"Elevation bands flowlines" are another way to transform a glacier into a
+"Elevation bands flowlines" method is another way to transform a glacier into a
```
Comment (fmaussion) [inline docs/flowlines.rst:169]:
```suggestion
The "elevation bands flowlines" method is another way to transform a glacier into a
```

File: docs/flowlines.rst [hunk 3]
```diff
@@ -186,15 +189,15 @@ bin size and slope. For OGGM, we then convert this first flowline to a regularly
...
+The resulting glacier flowlines can be understood as a "bulk" representation
 of the glacier, representing the average size and slope of each
 elevation band.
 
 Compatibility within the OGGM framework
 ---------------------------------------
 
 Both methods are creating a "1.5D" glacier. After computation,
-**both representations are strictly equivalent** for the inversion and
+**both representations are strictly equivalent** for the ice thickness inversion and
```
Comment (fmaussion) [inline docs/flowlines.rst:200]:
```suggestion
**both representations are programmatically equivalent** for the ice thickness inversion and
```

File: docs/ice-dynamics.rst [hunk 4]
```diff
@@ -1,7 +1,7 @@
-Ice dynamics
+Ice dynamic
```
Comment (fmaussion) [inline docs/ice-dynamics.rst:1]:
```suggestion
Ice dynamics
```

I think the plural form is correct https://en.wikipedia.org/wiki/Ice-sheet_dynamics

File: docs/ice-dynamics.rst [hunk 5]
```diff
@@ -40,7 +40,9 @@ basal shear stress :math:`\tau`:
 
     u = u_d + u_s = f_d h \tau^n + f_s \frac{\tau^n}{h}
 
-The second term is to account for basal sliding, see e.g. [Oerlemans_1997]_ or
+wehere u_d and u_s are respectively the speed of ice coming from its deformation and sliding
+(`n` is equal to 3). Thus, the first term is for ice deformations and the second term
```
Comment (fmaussion) [inline docs/ice-dynamics.rst:44]:
```suggestion
(`n` is equal to 3). Thus, the first term is for ice deformation and the second term 
```

File: docs/ice-dynamics.rst [hunk 6]
```diff
@@ -50,16 +52,17 @@ temperature‐dependent creep parameter :math:`A`:
 
     f_d = \frac{2 A}{n + 2}
 
-The basal shear stress :math:`\tau` depends e.g. on the geometry of the bed
+The basal shear stress :math:`\tau` depends for examle on the geometry of the bed
```
Comment (fmaussion) [inline docs/ice-dynamics.rst:55]:
```suggestion
The basal shear stress :math:`\tau` depends for example on the geometry of the bed
```

File: docs/ice-dynamics.rst [hunk 7]
```diff
@@ -121,7 +124,7 @@ Parabolic
 
 
 Parabolic shape with one degree of freedom, which makes it particularly
-useful for the bed inversion: if :math:`S` and :math:`w` are known:
+useful for the bed inversion .If :math:`S` and :math:`w` are known:
```
Comment (fmaussion) [inline docs/ice-dynamics.rst:127]:
```suggestion
useful for the bed inversion. If :math:`S` and :math:`w` are known:
```

File: docs/index.rst [hunk 8]
```diff
@@ -7,7 +7,7 @@ future mass-balance, volume and geometry of (almost) any glacier in the world,
 in a fully automated and extensible workflow.
 
 The model accounts for glacier geometry (including contributory branches) and
-includes an explicit ice dynamics module.  We rely exclusively on publicly
+includes an explicit ice dynamic module and a calving parametrization.  We rely exclusively on publicly
```
Comment (fmaussion) [inline docs/index.rst:10]:
```suggestion
includes an explicit ice dynamics module and a calving parametrization.  We rely exclusively on publicly
```

File: docs/inversion.rst [hunk 9]
```diff
@@ -85,13 +85,13 @@ there is no "optimum" :math:`f_{inv}` parameter in the model. There is a high
 uncertainty in the "true" :math:`A` parameter as well as in all other processes
 affecting the ice thickness. Therefore, we cannot make any recommendation for
 the "best" parameter. Global sensitivity analyses show that the default value
-is a good compromise ([Maussion_etal_2019]_),
-but very likely leads to overestimated ice volume.
+is a good compromise [Maussion_et al_2019]_,
```
Comment (fmaussion) [inline docs/inversion.rst:88]:
```suggestion
is a good compromise [Maussion_et_al_2019]_,
```

**Review by fmaussion — COMMENTED** [review]
Thanks so much! That's great. I just made some suggestions, I think you can accept them via github.

File: docs/ice-dynamics.rst (hunk 4 above)
Comment (nchampollion) [reply docs/ice-dynamics.rst:1]: Yes you're right. I though this opposite.

**Comment by nchampollion** [comment]
Yes I accepted the suggestions directly on GitHub :-)

**Pull request merged**